ccextender.yaml

```diff
@@ -1,44 +1,312 @@
+    golang-private:
+      - golang-private-library
+  io:
+    query:
+      prompt: "Will you need [1] both an input stream and output stream, [2] just an input stream, [3] just an output stream, or [0] no message streams?"
+      exclude-if:
+        - golang-open
+        - python-open
+        - library
+    both-io:
+      - input
+      - output
+      - streamer-manager
+    input:
+      - input
+      - streamer-manager
+    output:
+      - output
+      - streamer-manager
 change-packs:
-  docker:
-    template-go-docker:
-  go-service:
-    template-go-layout:
-    template-go-lint:
-  oss:
-    template-oss:
+  golang-base:
+    template-base:
+    template-go-base:
+  python-base:
+    template-base:
+    template-python-base:
+  library:
+  service:
+  golang-open-library:
+    # Currently nothing needed here
+  python-open-library:
+    # Currently nothing needed here
+  golang-private-library:
+    template-go-private-library:
+  python-private-library:
+    # Currently nothing needed here
+  python-open-service:
+    template-makefile:
+        sdcli_lang: python
+    template-python-open-service:
+    template-docker:
+  python-private-service:
+    template-makefile:
+        sdcli_lang: python
+    template-python-private-service:
+  golang-open-service:
+    template-go-open-service:
+    template-docker:
+    template-makefile:
+      sdcli_lang: go
+  golang-private-service:
+    template-go-private-service:
+    template-makefile:
+      sdcli_lang: go
+  input:
+    template-inbound:
+    template-micros:
+      service_descriptor: "
+                           \ benthos-input:\n
+                           \   image: ${BENTHOS_INPUT_ARTIFACT_NAME}\n
+                           \   tag: ${ARTIFACT_VERSION}\n
+                           \   deployRestartCount: 3\n
+                           \   ports:\n
+                           \     - \"4195:4195\"\n
+                           \   links:\n
+                           \     - gateway-incoming-private\n
+                           \     - statsd-service-sidecar\n
+                           \ gateway-incoming-private:\n
+                           \   image: ${GATEWAY_INCOMING_PRIVATE_ARTIFACT_NAME}\n
+                           \   tag: ${ARTIFACT_VERSION}\n
+                           \   deployRestartCount: 3\n
+                           \   links:\n
+                           \     - app\n
+                           \     - statsd-service-sidecar"
+      docker_compose: "
+                       \ benthos-input:\n
+                       \   build:\n
+                       \     context: .\n
+                       \     dockerfile: benthos-input.Dockerfile\n
+                       \     args:\n
+                       \       REGISTRY: ${REGISTRY}\n
+                       \   image: ${IMAGE}-benthos-input:${TAG}\n
+                       \   environment:\n
+                       \     APP_INPUT_ENDPOINT: \"http://gateway-inbound-private:8082/event\"\n
```

Comment:
Task: `gateway-inbound-private` should be `gateway-incoming-private`

Reply:
Oops, good catch. Fixing now